.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+<!-- Thank you for submitting a pull request! Find more information in our development guide at https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md and contact us on #python-client in Slack. -->
+<!-- Provide an overview of the PR for maintainers and reviewers. -->
+## Description
+
+
+
+<!-- Provide a user-friendly explanation, keep it brief if it isn't user-visible. -->
+## Non-technical explanation
+
+
+
+<!-- Mark the appropriate option with an (x). Propose a release note if you can. -->
+## Release notes
+
+( ) This is not user-visible and no release notes are required.
+( ) Release notes are required, please propose a release note for me.
+( ) Release notes are required, with the following suggested text:
+
+```markdown
+# Section
```

Comment:
This might not be needed in the python client since the release notes are probably just one list of items without sections/headers.

Reply:
SQLAlchemy, DB-API, General improvements? But yes - it feels overkill for now. We can add it back if needed.

Reply:
Should I remove a whole markdown section then?

Reply:
I think Manfred just means the `# Section` header. The markdown template is useful to have.

Reply:
Done